include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -4630,6 +4630,21 @@ struct OrtApi {
    * \snippet{doc} snippets.dox OrtStatus Return Value
    */
   ORT_API2_STATUS(KernelInfoGetAllocator, _In_ const OrtKernelInfo* info, _In_ OrtMemType mem_type, _Outptr_ OrtAllocator** out);
+
+  /** \brief Set external initializer file buffer
+   *
+   * \param[in] session options
+   * \param[in] external_file_names Array of null terminated UTF-8 encoded strings of the external initializer file names.
```

Comment:
Is this expected to be exact match to the file path defined in the ONNX model?

Reply:
the file path can start with ./, .\\, no problem